spec/features/practice_editor/implementation/practice_resources_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+describe 'Practice editor', type: :feature, js: true do
+  before do
+    @admin = User.create!(email: 'toshiro.hitsugaya@soulsociety.com', password: 'Password123', password_confirmation: 'Password123', skip_va_validation: true, confirmed_at: Time.now, accepted_terms: true)
+    @practice = Practice.create!(name: 'A public practice', slug: 'a-public-practice', approved: true, published: true, tagline: 'Test tagline')
+    @admin.add_role(User::USER_ROLES[0].to_sym)
+  end
+
+  describe 'Resources Page' do
+    before do
```

Comment:
do we want to have more tests here... where we are actually adding resources, files, and links?

Reply:
yeah.. i started writing tests for this.. when I noticed the other broken tests yesterday...  Alex wanted this up on Staging tonight.. so maybe we can still push it in the morning and then just add another story to write more tests for it?

Reply:
oh ok! creating a separate story for writing tests sound good to me so we can get this on staging. let me know if I can help write any tests! :)

Reply:
thoughts on this, @JoshingYou1?